Cheat Engine/bin/celua.txt

```diff
@@ -1826,11 +1828,23 @@ properties
   IsReadable: boolean - Set to false if record contains an unreadable address. NOTE: This property will not be set until the value property is accessed at least once. (ReadOnly)
 
   Options: String set - a string enclosed by square brackets filled with the options seperated by a comma. Valid options are: moHideChildren, moActivateChildrenAsWell, moDeactivateChildrenAsWell, moRecursiveSetValue, moAllowManualCollapseAndExpand, moManualExpandCollapse
+  
+  DropDownLinked: boolean - if dropdown list refers to list of another memory record eg. (memrec name)
+  DropDownLinkedMemrec: string - Description of linked memrec or emptystring if not linked
+  DropDownList : StringList - self explanatory (returns new list each time)

```

Comment:
>(returns new list each time)

iirc, it will return the same list each time. Try this script few times for first memrec with DropDownList
```Lua
return userDataToInteger(AddressList[0].DropDownList)
```

Reply:
yeah it's returning the same one...I could have sworn I tested that and that's why I made the note though lol
Perhaps I got confused with a linked memrec returning different ones... anyways, glad you caught that, new commit coming up :)

Reply:
btw, is there a reason that DropDownLinkedMemrec is the description of the linked memrec rather than the memrec object or at least the ID?